src/background/services/paymentSession.ts

```diff
@@ -120,76 +118,34 @@ export class PaymentSession {
     if (this.active) return
     this.active = true
 
-    const data = await this.storage.get(['token', 'walletAddress'])
-
-    let token = data.token
-    const walletAddress = data.walletAddress
-
-    if (token == null || walletAddress == null) {
-      return
-    }
-
     await this.setIncomingPaymentUrl()
 
     while (this.active) {
       let quote: Quote | undefined
       let outgoingPayment: OutgoingPayment | undefined
 
       try {
-        if (!quote) {
-          quote = await this.openPaymentsService.client!.quote.create(
-            {
-              url: walletAddress.resourceServer,
-              accessToken: token.value
-            },
-            {
-              method: 'ilp',
-              receiver: this.incomingPaymentUrl,
-              walletAddress: walletAddress.id,
-              debitAmount: {
-                value: this.amount,
-                assetScale: walletAddress.assetScale,
-                assetCode: walletAddress.assetCode
-              }
-            }
-          )
+        // Quote can be removed once the Test Wallet upgrades to alpha-10.
+        // We will be able to create an outgoing payment with an incoming payment,
+        // making the quoting unnecessary through OP.
+        //
+        // Note: Under the hood, Rafiki is still quoting.
+       if (!quote) {
```

Comment:
Why do we have this `if (!quote)`, which is always true? We just defined it as such above.

Reply:
But we've:
```js
while (this.active) {
      let quote: Quote | undefined
      try {
```

If `quote` is not created, `this.openPaymentsService.createQuote` will throw, right? That means, `quote` isn't assigned and we get into catch block. `quote` is not used outside of that try block from what I see.